### PR TITLE
File Details Page

### PR DESCRIPTION
**Pull request by BruceJi7**

### CODE REVIEW

File: src/pages/FileDetails/AdditionalDetailsCell/AdditionalDetailsCell.module.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.Cell {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  // margin: 10px;
```
Comment (ivanms1) [inline src/pages/FileDetails/AdditionalDetailsCell/AdditionalDetailsCell.module.scss:5]: remove plz

File: src/pages/FileDetails/AdditionalDetailsCell/TagsBox/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import styles from "./TagsBox.module.scss";
+
+function TagsBox({ tags }: { tags: Array<string> }) {
```
Comment (ivanms1) [inline src/pages/FileDetails/AdditionalDetailsCell/TagsBox/index.tsx:5]:
```suggestion
function TagsBox({ tags }: { tags: string[] }) {
```

File: src/pages/FileDetails/AdditionalDetailsCell/UserDetails/index.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import Button from "../../../../components/Button";
+
+import styles from "./UserDetails.module.scss";
+import CopyIcon from "../../../../assets/icons/copy.svg";
+import NoAvatar from "../../../../assets/icons/no-avatar.svg";
+import useGetImage from "../../../../hooks/useGetImage";
```
Comment (ivanms1) [inline src/pages/FileDetails/AdditionalDetailsCell/UserDetails/index.tsx:8]: import order

File: src/pages/FileDetails/AdditionalDetailsCell/UserDetails/index.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+function Avatar({ avatar }: { avatar: string }) {
+  const { data: avatarImgSrc } = useGetImage(avatar);
+
+  if (avatar) {
+    return <img src={avatarImgSrc} className={styles.Avatar} />;
+  } else {
```
Comment (ivanms1) [inline src/pages/FileDetails/AdditionalDetailsCell/UserDetails/index.tsx:15]: else not needed

File: src/pages/FileDetails/AdditionalDetailsCell/index.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import UserDetails from "./UserDetails";
+import Description from "./Description";
+import TagsBox from "./TagsBox";
+
+import styles from "./AdditionalDetailsCell.module.scss";
+import { FileProps } from "../../../types";
```
Comment (ivanms1) [inline src/pages/FileDetails/AdditionalDetailsCell/index.tsx:8]: order

File: src/pages/FileDetails/AdditionalDetailsCell/UserDetails/index.tsx (hunk 3 above)
Comment (BruceJi7) [reply src/pages/FileDetails/AdditionalDetailsCell/UserDetails/index.tsx:8]: yep

File: src/pages/FileDetails/FileDetails.module.scss [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  background-color: var(--bg);
+  background-color: white;
+  border-radius: 5px;
+  box-sizing: border-box;
+  display: flex;
+  height: calc(596px - 40px);
+  justify-content: center;
+  padding: 40px;
+  position: relative;
+  // width: 740px;
```
Comment (ivanms1) [inline src/pages/FileDetails/FileDetails.module.scss:11]: remove

File: src/pages/FileDetails/MainCell/LikeBar/LikeBar.module.scss [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.LikeNumber {
+  color:var(--grey-button-color);
+  margin-left:7px;
+
+}
+
+.Uploaded {
+  font-size: 8pt;
```
Comment (ivanms1) [inline src/pages/FileDetails/MainCell/LikeBar/LikeBar.module.scss:24]: ?

File: src/pages/FileDetails/MainCell/MainCell.module.scss [hunk 8]
```diff
@@ -0,0 +1,32 @@
+.Cell {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  // margin: 10px;
```
Comment (ivanms1) [inline src/pages/FileDetails/MainCell/MainCell.module.scss:5]: remove please

File: src/pages/FileDetails/MainCell/index.tsx [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+import FileProperties from "./FileProperties";
+
+import useCurrentUser from "../../../hooks/useCurrentUser";
+import useDownloadFile from "../../../hooks/useDownloadFile";
+
+import { FileProps } from "../../../types";
+import trunc from "../../../helpers/trunc";
+
+import styles from "./MainCell.module.scss";
+import useGetImage from "../../../hooks/useGetImage";
```
Comment (ivanms1) [inline src/pages/FileDetails/MainCell/index.tsx:17]: order

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**